metron-extensions/README.md

```diff
@@ -0,0 +1,51 @@
+# Metron Extensions
```

Comment:
Your new extension mechanism seems like a core component for the platform.  Should this live under `metron-platform` instead of being top-level?